Middleware/src/main/java/ca/concordia/ngci/tools4cities/middleware/middleware/IProducer.java

```diff
@@ -9,5 +9,8 @@ public interface IProducer<E> {
 	void addObserver(final IConsumer<E> aConsumer);
 	
 	void notifyObservers(final List<E> results);
+	
+	// I defined this method because I use it for my local csv file
```

Comment:
Remove commented methods. If you want to keep some methods only locally, keep the code in another branch. You can always run git checkout -b nameofbranch to create a copy of your current work.